format.go

```diff
@@ -16,7 +13,7 @@ const (
 type Units uint
 
 const (
-	UnitNone Units = iota
```

Comment:
Thank you for your contribution! Just curios what is your use case of using mpb?